apps/web/src/pages/Project/Project.tsx

```diff
@@ -169,44 +199,61 @@ function Project() {
               </div>
             </div>
           </div>
-          <LikeButton project={data?.project} />
+          <div className='flex flex-row items-start gap-[50px] '>
+            <LikeButton project={data?.project} />
+          </div>
         </div>
-        {isProjectOwner && (
-          <div className='flex gap-5 mr-0 ml-auto'>
-            <Link href={`/project-edit/${router?.query?.id}`}>
-              {t('common:edit')}
-            </Link>
-
-            <Button
-              variant='ghost'
-              className='text-red-500'
-              onClick={() => setOpenDeleteModal(true)}
-            >
-              {t('common:delete')}
-            </Button>
 
-            <Modal
-              open={openDeleteModal}
-              onClose={() => setOpenDeleteModal(false)}
-              modalClassName='bg-grey-dark flex flex-col justify-center p-12 max-lg:h-[50vh]'
+        <div className='flex items-center gap-5 mr-0 ml-auto'>
+          {isProjectOwner ? (
+            <div className='flex items-center gap-5'>
+              <Link href={`/project-edit/${router?.query?.id}`}>
+                {t('common:edit')}
+              </Link>
+
+              <Button
+                variant='ghost'
+                className='text-red-500'
+                onClick={() => setOpenDeleteModal(true)}
+              >
+                {t('common:delete')}
+              </Button>
+
+              <Modal
+                open={openDeleteModal}
+                onClose={() => setOpenDeleteModal(false)}
+                modalClassName='bg-grey-dark flex flex-col justify-center p-12 max-lg:h-[50vh]'
+              >
+                <p className='mb-10 text-2xl font-semibold'>
+                  {t('project:are-you-sure')}
+                </p>
+                <div className='flex justify-between'>
+                  <Button
+                    variant='secondary'
+                    onClick={() => deleteProjectClick(data?.project?.id)}
+                  >
+                    {t('common:yes')}
+                  </Button>
+                  <Button onClick={() => setOpenDeleteModal(false)}>
+                    {t('common:no')}
+                  </Button>
+                </div>
+              </Modal>
+            </div>
+          ) : (
+            <button
+              onClick={() =>
+                isLoggedIn
+                  ? setOpenReportModal(true)
+                  : setIsLoginModalOpen(true)
+              }
+              title='Report Project'
+              className='w-[40px] h-[40px] flex items-center justify-center rounded-[10px] bg-grey-dark cursor-pointer'
```

Comment:
use the tailwind presets as much as possible
```suggestion
              className='w-10 h-10 flex items-center justify-center rounded-[10px] bg-grey-dark cursor-pointer'
```

Reply:
ok